docs/running-on-kubernetes.md

```diff
@@ -393,6 +393,27 @@ RBAC authorization and how to configure Kubernetes service accounts for pods, pl
 [Using RBAC Authorization](https://kubernetes.io/docs/admin/authorization/rbac/) and
 [Configure Service Accounts for Pods](https://kubernetes.io/docs/tasks/configure-pod-container/configure-service-account/).
 
+## Spark Application Management
+
+Like Mesos and Standalone, Kubernetes provides simple application management via th spark-submit cli tool in cluster mode.
+Users can kill a job by providing the submission id that is printed when submitting their job.
+The submission id follows the following format namespace:driver-pod-name and is similar to the driverId found in Standalone mode.
```

Comment:
Nit: put `namespace:driver-pod-name` in a `` block.

Reply:
ok